app/models/User.js

```diff
@@ -170,6 +170,24 @@ const UserSchema = new mongoose.Schema({
       }
     }
   ],
+  ticketNotifications: [
+    {
+      heading: {
+        type: String
+      },
+      content: {
+        type: String
+      },
+      tag: {
+        type: String
+      },
+      createdAt: {
```

Comment:
required should not be there, it's default behaviour of mongoose , same for update